editor/plugins/tiles/atlas_merging_dialog.cpp

```diff
@@ -52,15 +52,15 @@ void AtlasMergingDialog::_generate_merged(Vector<Ref<TileSetAtlasSource>> p_atla
 		// Compute the new texture region size.
 		Vector2i new_texture_region_size;
 		for (int source_index = 0; source_index < p_atlas_sources.size(); source_index++) {
-			Ref<TileSetAtlasSource> atlas_source = p_atlas_sources[source_index];
+			const Ref<TileSetAtlasSource> &atlas_source = p_atlas_sources[source_index];
 			new_texture_region_size = new_texture_region_size.max(atlas_source->get_texture_region_size());
```

Comment:
```suggestion
			new_texture_region_size = new_texture_region_size.max(p_atlas_sources[source_index]->get_texture_region_size());
```

Reply:
I'm biased against this change -- the same `atlas_source = <...>` pattern is used in other places.